config/plugins/visualizations/charts/static/repository/utilities/jobs.js

```diff
@@ -43,7 +43,7 @@ define( [ 'utilities/utils' ], function( Utils ) {
                     }
                     chart.state( 'failed', 'This visualization requires the \'' + parameters.tool_id + '\' tool. Please make sure it is installed. ' + message );
                     error && error();
-                          
```

Comment:
Personally I'd much rather see non-substantive changes to unrelated files as a separate easy-to-review PR, instead of lumped into something else.